### PR TITLE
test(dir_generation): Added more content to the dir generator

### PR DESCRIPTION
**Pull request by MartinFillon**

### CODE REVIEW

File: Justfile [hunk 1]
```diff
@@ -266,6 +268,76 @@ gen_test_dir:
...
+    # BEGIN test_size
+    mkdir -p size
+    touch size/1M --date=@0
+    dd if=/dev/zero of=size/1M bs=1 count=0 seek=1M
+    touch size/1K --date=@0
+    dd if=/dev/zero of=size/1K bs=1 count=0 seek=1K
+    touch size/1B --date=@0
+    dd if=/dev/zero of=size/1B bs=1 count=0 seek=1
+    touch size/69 --date=@0
+    dd if=/dev/zero of=size/69 bs=1 count=0 seek=69
```
Comment (cafkafk) [inline Justfile:308]: \>_>
```suggestion
    dd if=/dev/zero of=size/1337 bs=1 count=0 seek=69
```

**Review by cafkafk — CHANGES_REQUESTED** [review]
Oh nice, glad to see work on tests :tada:

File: Justfile (hunk 1 above)
Comment (MartinFillon) [reply Justfile]: can't use your suggestion it is not complete but changed the file name and size

**Review by cafkafk — APPROVED** [review]
LGTM, thanks for adding this :tada:

**Pull request merged**